avaframe/com1DFAPy/com1DFA.py

```diff
@@ -47,37 +50,46 @@
 log.info('MAIN SCRIPT')
 log.info('Current avalanche: %s', avalancheDir)
 
-cfg = cfgUtils.getModuleConfig(DFAtls)['GENERAL']
-cfgFull = cfgUtils.getModuleConfig(DFAtls)
+# Load configuration
+cfg = cfgUtils.getModuleConfig(DFAtls)
+cfgGen = cfg['GENERAL']
 
+# for timing the sims
 startTime = time.time()
+
+
+#+++++++++Inputs++++++++++++++++++++++++
 # ------------------------
-# fetch input data
-demFile, relFiles, entFiles, resFile, flagEntRes = gI.getInputData(avalancheDir, cfgFull['FLAGS'], flagDev=True)
+# fetch input data - dem, release-, entrainment- and resistance areas
+demFile, relFiles, entFiles, resFile, flagEntRes = gI.getInputData(avalancheDir, cfg['FLAGS'], flagDev=True)
 demOri = IOf.readRaster(demFile)
+# derive line from release area polygon
 releaseLine = shpConv.readLine(relFiles[0], 'release1', demOri)
 dem = copy.deepcopy(demOri)
 dem['header'].xllcenter = 0
 dem['header'].yllcenter = 0
 dem['header'].xllcorner = 0
 dem['header'].yllcorner = 0
-
 # ------------------------
-# process release to get it as a raster
+# process release info to get it as a raster
 relRaster = DFAtls.prepareArea(releaseLine, demOri)
 relTh = 1
 # could do something more advanced if we want varying release depth
 relRasterD = relRaster * relTh
 
+
+#+++++++++INITIALIZE SIMULAITON++++++++++++++++++++++++
 massPart = 1250  # [200, 100, 50, 25, 10, 7.5, 5]
```

Comment:
Either as feature parameter in here (top of file) or as cfg- parameter

Reply:
this was used before, not just in this feature.. but it is actually defined in the com1DFAtoolsCfg.ini